Curs12/C/03-params_sum.c

```diff
@@ -0,0 +1,29 @@
+/* Programarea Calculatoarelor CA, 2022-2023
+ *
+ *	Suma elementelor parametrilor liniei de comanda
+ *
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main(const int argc, const char **argv)
+{
+	int s = 0;
+
+	for (int i = 1; i < argc; i++)
+		s += atoi(argv[i]);
+	printf("%d\n", s);
+
+	return 0;
+}
+
+/*
+ *	compile: gcc 03-params_sum.c -o params_sum
+ *	run: ./params_sum 1 2 3 4 5
```

Comment:
Maybe check if the CLI params (1 <= i <= argc - 1) are numbers ? (aka defensive programming before using atoi())

Reply:
yes, as this would be an invalid input